libraries/hermes/src/Rest/Swagger.ts

```diff
@@ -42,9 +42,13 @@ export class SwaggerGenerator {
   addRouteSwaggerDocumentation(route: ConduitRoute) {
     const method = this._extractMethod(route.input.action);
     let serviceName = route.input.path.toString().replace('/hook', '').slice(1);
-    serviceName = serviceName.substr(0, serviceName.indexOf('/'));
-    if (serviceName.trim() === '') {
+    let prefix = serviceName.substring(0, serviceName.indexOf('/'));
```

Comment:
Make this a constant.

Reply:
Actually no, the previous line is what prefix should be initialized as.
Service name should be declared early, but assigned an initial value in the if / else blocks below.